src/commands/createFunctionApp/stacks/getStackPicks.ts

```diff
@@ -176,3 +197,9 @@ function removeHiddenStacksAndProperties(stacks: FunctionAppStack[]): void {
         }
     }
 }
+
+export function compareDates(eolDate: string, sixMonthsDate: Date): boolean {
```

Comment:
I think it'd be best if this function was a bit more specific to our use case. Something like `shouldShowEolWarning`. Will allow you to hide all of the EOL specific logic in there instead of before calling the function..